magento/magento2
# Resolved 404 issue of crdit memo if used wrong id in url

Resolved 404 issue of crdit memo if used wrong id in url


### Description (*)
Fatal error page occurs if use wrong creditmemo id passed in url


   ###Manual testing scenarios (*)

    1.Create An Order In Magento

    2.Create invoice and creditmemo of that Order

    3.View the created creditmemo

    4.Now change the creditmemo id from url(use that id which does not exist) and run the ur

###Expected result (*)
A proper error message should come without any error

###Actual Result (*)
Fatal error page occurs

### Contribution checklist (*)
